mailpoet/tests/performance/tests/segments-create-custom.js

```diff
@@ -121,18 +126,17 @@ export async function segmentsCreateCustom() {
     });
 
     // Save the segment
-    const calculatedMessage =
-      "//div[@class='mailpoet-form-field'].//span[starts-with(text(),'This segment has')]";
-    await calculatedMessage.waitFor();
-    await focusAndClick(page, 'div.mailpoet-form-actions > button');
-    await page.waitForSelector('[data-automation-id="filters_all"]', {
+    await waitForSelectorToBeClickable(page, 'button[type="submit"]');
+    await focusAndClick(page, 'button[type="submit"]');
```

Comment:
The `button[type="submit"]` selector is too broad: if there's more than one submit button on the page (which is pretty common), it may select a wrong button and the test will fail. It's better to be more specific: add a unique `data-automation-id` attribute to the button, specify a parent form class/ID etc. I see the latter was done previously, was it unsuitable for you for some reason?

Reply:
I didn't expect any new submit button on this page and wanted to make more readable selector for button submit.
I just reverted it to the old selector, it works fine with it. Thanks for the review!

Reply:
I updated it also in the other segments test.